src/hotkey.rs

```diff
@@ -48,3 +53,40 @@ pub fn spawn_hotkey_thread() {
         }
     });
 }
+
+fn compile_modifiers(activators: &Vec<String>) -> u32 {
+    let mut code: u32 = 0;
+    for key in activators.iter() {
+        match key.as_str() {
+            "ALT"   => code = code | MOD_ALT as u32,
+            "CTRL"  => code = code | MOD_CONTROL as u32,
+            "SHIFT" => code = code | MOD_SHIFT as u32,
+
+            _ => unsafe {
+                report_and_exit("Invalid hotkey: Unidentified modifier in hotkey combination.")
+            },
+        }
+    }
+    code
+}
+
+unsafe fn get_vkcode(key_char: char) -> u32 {
+    let keyboard_layout = GetKeyboardLayout(0);
+    let vk_code = VkKeyScanExA(key_char as i8, keyboard_layout).to_be_bytes();
```

Comment:
Use the `VkKeyScanExW` function... W version of functions uses Unicode, which should be much more compatible w/ Rust